tests/cpp/kvstore/gpu_topology_test.cc

```diff
@@ -71,11 +67,12 @@ void TestComputeTreesRandomized(int num_gpus, float alpha, int backtrack,
   bool satisfied = false;
   std::vector<float> W(num_gpus*num_gpus, 0.f);
   int depth = mxnet::kvstore::ComputeDepth(num_gpus);
-  while (!satisfied) {
-    float k = dis(*gen);
-    std::fill(W.begin(), W.end(), 0.f);
-    GenerateMatrix(&W, num_gpus, k, gen);
-    satisfied = IsSatisfactory(W, num_gpus, depth);
+  GenerateMatrix(&W, num_gpus, gen);
+  satisfied = IsSatisfactory(W, num_gpus, depth);
+  if (mxnet::kvstore::kLogTree && !satisfied) {
```

Comment:
Previously it was trying multiple times - we want to stop doing it with this change?

Reply:
yes, it's of no use.